addons/sourcemod/scripting/freak_fortress_2.sp

```diff
@@ -4405,9 +4406,41 @@ public OnClientPutInServer(client)
 
 public OnClientDisconnect(client)
 {
-	if(Enabled && IsClientInGame(client) && IsPlayerAlive(client) && CheckRoundState()==1)
+	if(Enabled)
 	{
-		CreateTimer(0.1, CheckAlivePlayers);
+		if(IsBoss(client) && cvarPreroundBossDisconnect)
```

Comment:
This check is bad for two reasons:
1. It won't force a team win if `cvarPreroundBossDisconnect` is false and the round has already started
2. It'll always return true since you're comparing a `Handle` to true
```sourcepawn
if(IsBoss(client))
{
    if(CheckRoundState()==1)
    {
        ForceTeamWin(OtherTeam);
    }

    if(GetConVarBool(cvarPreroundBossDisconnect))  //You might want to cache this instead, your choice
    {
        //Code
    }
}
```